shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RouteCardData.kt

```diff
@@ -650,7 +651,32 @@ data class RouteCardData(
             pinnedRoutes: Set<String>,
             context: Context,
         ): List<RouteCardData>? =
-            withContext(Dispatchers.Default) {
+            routeCardsForStopList(
+                stopIds,
+                globalData,
+                sortByDistanceFrom,
+                schedules,
+                predictions,
+                alerts,
+                now,
+                pinnedRoutes,
+                context,
+                Dispatchers.Default,
+            )
+
+        suspend fun routeCardsForStopList(
+            stopIds: List<String>,
+            globalData: GlobalResponse?,
+            sortByDistanceFrom: Position?,
+            schedules: ScheduleResponse?,
+            predictions: PredictionsStreamDataResponse?,
+            alerts: AlertsStreamDataResponse?,
+            now: Instant,
+            pinnedRoutes: Set<String>,
+            context: Context,
+            coroutineDispatcher: CoroutineDispatcher,
```

Comment:
Why declare this as a separate function rather than just a new parameter with a default argument in the original function?